src/registrar/models/user.py

```diff
@@ -39,6 +51,19 @@ def __str__(self):
         else:
             return self.username
 
+    def block_user(self):
+        self.status = "ineligible"
+        self.save()
+
+    def unblock_user(self):
+        self.status = None
+        self.save()
+
+    def is_blocked(self):
+        if self.status == "ineligible":
+            return True
+        return False
```

Comment:
```suggestion
    def is_blocked(self):
        return self.status == self.INELIGIBLE
```
You can simplify this function by moving your if statement into your return